circleguard/draw.py

```diff
@@ -55,8 +56,13 @@ def run(self):
         data1 = Replay.resample(data1, fps)
         data2 = Replay.resample(data2, fps)
 
+        flip1 = Mod.HardRock.value in [mod.value for mod in self.replay1.enabled_mods]
```

Comment:
This logic is duplicated, here and in comparer.py, so it's probably about time we made a utils.py for stuff like this and resampling. Let's do it in another pr though.